docker-compose/README.md

```diff
@@ -1,19 +1,31 @@
-# docker-compose for Liquidity Provider Server
+# Docker Setup for Liquidity Provider Server
 
-This docker-compose files can be used to quickly spin up an environment with the server and its
-dependant services (`bitcoind` and `rskj`) for either `regtest` or `testnet`
+The provided docker-compose files can be used to quickly spin up an environment with the Liquidity Provider Server and its dependent services (`bitcoind` and `rskj`) for either `regtest` or `testnet`.
```

Comment:
I would clarify in this section that it is not the same file for both, we had this confusion with Systema, because the regtest env requires a regtest federation and a localstack so the setup is a bit different, for example, only the local env has the script to spin up the environment, testnet or mainnet should be done directly with the compose files